en/appendices/4-3-migration-guide.rst

```diff
@@ -97,6 +107,18 @@ Http
 - The ``CspMiddleware`` now sets the ``cspScriptNonce`` and ``cspStyleNonce``
   request attributes which streamlines the adoption of strict
   content-security-policy rules.
+  
+Log
+---
```

Comment:
I can add a section to the logging docs to give a guide on how to use formatted and build new ones.

Reply:
Should I reduce the migration notes here? I don't know if it's overly verbose.